usr/sbin/pkcshsm_mk_change/pkcshsm_mk_change.c

```diff
@@ -13,10 +13,8 @@
  * secure keys for a concurrent HSM master key change.
```

Comment:
Since `pkcscca` is still disabled on AIX, the only way to perform a secure key re-enciphering after a CCA master key change is to use concurrent MK change using the `pkcshsm_mk_change` tool. 

However, `pkcshsm_mk_change` requires the user to specify an APQN (`<card>.<domain>`)  or a list of APQNs of which the master keys have been changed. Unfortunately, the term APQN is not really known/used by AIX. In fact, the CCA token internally always uses card 0 and domain 0 on AIX. 

This should still work with `pkcshsm_mk_change` but the user must specify `0.0` as APQN on AIX, so that the CCA token recognizes that it is affected by the MK change. 

Maybe you should add something to the man page of `pkcshsm_mk_change` to describe the special usage on AIX. Maybe also to the usage text of the `pkcshsm_mk_change` tool?